src/pages/dashboard/Routes.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Routes = route => {
+  return <p>{route} is the route for the driver</p>;
+};
```

Comment:
for routes that have url params ending in `/:somethingId` to get the actual id use the `useParams` hook from `react-router-dom` as shown in [this example](https://v5.reactrouter.com/web/example/url-params)